### PR TITLE
merge feature top page to develop

### PR DESCRIPTION
**Pull request by Shiho317**
Hi, guys. I just merged feature-top-page-responsive branch. In this branch, I added scroll function. 
So I want to check that scroll function is working well or not after merged. ( click menu button -> jump to specific section)
If you got something wrong, pls let me know. 
Thanks.

### CODE REVIEW

File: components/menu.js
Comment (shiv-chan) [inline components/menu.js]: スクリーンショット 2021-12-13 10 25 50
@Shiho317 probably you just followed the code that I wrote before but do you think the following HTML structure has more sense than the above?

```html
<ul>
  <li><a href="#home">HOME</a></li>
  <li><a href="#episodes">EPISODES</a></li>
  <li><a href="#about">ABOUT</a></li>
  <li><a href="#contact">CONTACT</a></li>
</ul>
```
If there is something wrong with this way, it's totally okay with the current one.

File: components/menu.js
Comment (shiv-chan) [inline components/menu.js:31]: Even though we have `spy={true}` I couldn't confirm the scroll spy was working.
I believe that this gives the element "active" selector, so let's add the style to "active".
You wanna specify this on `globals.css` file like the following.

```css
.active {
  color: white !important
}
```

File: components/menu.module.css
Comment (shiv-chan) [inline components/menu.module.css]: Don't we have an underline on mobile devices?
I think it's better to keep the consistency.

Also, I prefer a white underline 😄

File: components/menu.module.css
Comment (shiv-chan) [inline components/menu.module.css]:
```suggestion
  list-style: none;
  transition: color 0.1s ease;
  display: inline-block;
}
```
I realized that the color snaps and changes its colour, so I suggest putting `transition` like above.
Plus, I don't want each item to have a redundant/longer underline so I put `display: inline-block`.

It works like this:
menu-hover-actions

File: components/menu.module.css
Comment (shiv-chan) [inline components/menu.module.css]:
```suggestion
    border-bottom: 1px solid white;
```
Can we change it to white colour?

**Review by shiv-chan — APPROVED** [review]
Thank you for enhancing the menu and introducing a nice package, @Shiho317 👍 

I thought you've already merged `feature-top-page` to `develop`.
It's a bit hard to see where I've already reviewed and where I haven't.
Once one PR is approved, please merge that to the target branch.
(I believe that `feature-top-page-responsive` which includes `feature-top-page` has been approved.)
Then, create a new branch, let's say "feature-menu-scroll" and create a new PR.

I also left some comments and suggestions.
You can merge this branch to `develop`now but if you have time and apply my suggestions before merging it would be great 👏

File: components/menu.module.css
Comment (bluwakpa) [reply components/menu.module.css]: Agreed!

**Review by bluwakpa — APPROVED** [review]
You did a great job! Let's merge this.

**Pull request merged**
